lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoTypeDeclarationImpl.kt

```diff
@@ -45,6 +45,41 @@ internal class KoTypeDeclarationImpl private constructor(
 
     override val isNullable: Boolean by lazy { ktTypeReference.text.last() == '?' }
 
+    override val isKotlinType: Boolean by lazy {
+        if (isImportAlias()) {
+            false
+        } else {
+            basicTypes.any { sourceType.contains(it) } || collections.any { sourceType.contains(it) }
+        }
+    }
+
+    private val basicTypes: List<String> by lazy {
+        listOf(
+            "Byte",
+            "Short",
+            "Int",
+            "Long",
+            "Float",
+            "Double",
+            "UByte",
+            "UShort",
+            "UInt",
+            "ULong",
+            "UByteArray",
+            "UShortArray",
+            "UIntArray",
+            "ULongArray",
+            "Boolean",
+            "Char",
+            "String",
+            "Array",
+        )
+    }
+
+    private val collections: List<String> by lazy {
+        listOf("List", "Set", "Map", "MutableList", "MutableSet", "MutableMap")
```

Comment:
1. These Types are missing
<img width="585" alt="image" src="https://github.com/LemonAppDev/konsist/assets/530122/4bc33fcc-4473-457c-9f04-f12514dc8434">


2. Let's change the order, so mutable and normal collections are close to each other `"Map", ģMutableMap"...`